x-pack/plugins/infra/public/pages/metrics/layouts/host.ts

```diff
@@ -106,54 +145,83 @@ export const hostLayoutCreator: InfraMetricLayoutCreator = theme => [
       },
       {
         id: InfraMetric.hostNetworkTraffic,
-        label: 'Network Traffic',
+        label: i18n.translate('xpack.infra.hostMetrics.networkTrafficSectionLabel', {
+          defaultMessage: 'Network Traffic',
+        }),
         requires: ['system.network'],
         type: InfraMetricLayoutSectionType.chart,
         visConfig: {
           formatter: InfraFormatterType.bits,
           formatterTemplate: '{{value}}/s',
           type: InfraMetricLayoutVisualizationType.area,
           seriesOverrides: {
-            rx: { color: theme.eui.euiColorVis1, name: 'in' },
-            tx: { color: theme.eui.euiColorVis2, name: 'out' },
+            rx: {
+              color: theme.eui.euiColorVis1,
+              name: i18n.translate('xpack.infra.hostMetrics.visConfig.inName', {
+                defaultMessage: 'in',
+              }),
+            },
+            tx: {
+              color: theme.eui.euiColorVis2,
+              name: i18n.translate('xpack.infra.hostMetrics.visConfig.outName', {
+                defaultMessage: 'out',
+              }),
+            },
           },
         },
       },
     ],
   },
   {
     id: 'k8sOverview',
-    label: 'Kubernetes',
+    label: i18n.translate('xpack.infra.kubernetesMetrics.kubernetesLabel', {
+      defaultMessage: 'Kubernetes',
```

Comment:
According to [Glossary](https://docs.google.com/document/d/1e-lHAPAtxodP8U4gbwA3lw8FyhhW7M303--77nMpe8w/edit) we shouldn't translate this label.